AISKU/src/AISku.ts

```diff
@@ -92,7 +92,8 @@ const defaultConfigValues: IConfigDefaults<IConfiguration & IConfig> = {
     ),
     extensionConfig: cfgDfMerge<{[key: string]: any}>({
         ["AppInsightsCfgSyncPlugin"]: cfgDfMerge<ICfgSyncConfig>({
-            cfgUrl: CONFIG_ENDPOINT_URL
+            cfgUrl: CONFIG_ENDPOINT_URL,
+            syncMode: ICfgSyncMode.Receive
```

Comment:
change the AISKU default receive mode to "receive" (will call "updateCfg" to update current core.config), previous one is "broadcast" (which means only send events out but not receiving any incoming changes)